app/src/main/java/com/ikarabulut/GameSettings.java

```diff
@@ -14,14 +14,16 @@ public class GameSettings {
 
     private Scanner input = new Scanner(System.in);
 
+    private int intSelection;
```

Comment:
Looks like you're using `intSelection` as a variable to hold the number of rows wanted, and then the number of columns. I'd use two separate variables instead of overwriting the same variable. This just makes it harder to read.

Reply:
This was changed in the first PR `pretty-display` idk why I originally added this. At first I tried declaring this variable in my methods but it didn't look like it was scoped to the `try catch || if else` statements within that method which made no sense but I didn't want to stay blocked so I made it global.

After retrying it looks like it worked if I just declared the variable in the method.